src/parseGitmodulesDependencies.ts

```diff
@@ -0,0 +1,23 @@
+import { chunk } from 'lodash';
+import fse from 'fs-extra';
+
+export const parseGitmodulesDependencies = (): { [key: string]: string } => {
+  let iniAsStr;
+  try {
+    iniAsStr = fse.readFileSync('./.gitmodules', { encoding: 'utf8' });
+  } catch {
+    return {};
+  }
+  const matches = [...iniAsStr.matchAll(/(?:path|url) = (.*)/g)].map(regexMatch => regexMatch[1]);
+  if (matches.length == 0) return {};
+
+  // this assumes .gitmodules to be well formed, same amount of paths and urls
```

Comment:
If is not, shouldn't it throw an error describing the issue?
I see this as a NTH tho